app.json

```diff
@@ -4,6 +4,9 @@
     "postdeploy": "bin/review_app_setup"
   },
   "env": {
+    "NEW_RELIC_ENV": {
+      "required": true
+    },
```

Comment:
This does exactly what we _don't_ want: it copies the staging configuration. This will dump the review app data into the staging graph.

Can you set this to something else–"review" maybe?

Reply:
@leesharma fixed the issues, please check https://github.com/rubyforgood/playtime/pull/150/commits/9b3f9d49b03404313e2f7b685d432f2112b4456c
and really thanks a lot for the guidance ☺️